pkg/controller/certificates/signer/signer.go

```diff
@@ -305,3 +315,152 @@ func usagesToSet(usages []capi.KeyUsage) sets.String {
 	}
 	return result
 }
+
+type PCRSigningController struct {
```

Comment:
Why do we need this wrapper to the `PodCertificateRequestController`? The indirection pattern used in this package seems a bit confusing.

Reply:
I was just following the existing pattern used by all the different CSR signing controllers.  I'm not totally sure why it was written this way.

Anyway, I've dropped the built-in signers from the first PR so that we actually have a chance of landing some of this functionality in 1.33.